project/FrontEnd/collaborative_science_platform/lib/models/workspaces_page/workspaces.dart

```diff
@@ -3,20 +3,36 @@ import 'package:collaborative_science_platform/models/workspaces_page/workspaces
 class Workspaces {
   List<WorkspacesObject> workspaces;
   List<WorkspacesObject> pendingWorkspaces;
+  List<WorkspacesObject> reviewWorkspaces;
+  List<WorkspacesObject> pendingReviewWorkspaces;
   Workspaces({
     required this.workspaces,
     required this.pendingWorkspaces,
+    required this.pendingReviewWorkspaces,
+    required this.reviewWorkspaces,
   });
 
   factory Workspaces.fromJson(Map<String, dynamic> jsonString) {
```

Comment:
Your constructors are always well designed and easy to follow. Good job here and during all semester.